api/v1alpha1/integrationtestscenario_conversion.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"reflect"
+
+	"github.com/redhat-appstudio/integration-service/api/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+func (r *IntegrationTestScenario) SetupWebhookWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewWebhookManagedBy(mgr).
+		For(r).
+		Complete()
+}
+
+// Hub marks this type as a conversion hub.
+// ConvertTo converts this ITS to the Hub version (v1beta1).
+func (src *IntegrationTestScenario) ConvertTo(dstRaw conversion.Hub) error {
```

Comment:
Why is this called `ConvertTo` rather than something like `ConvertToHubVersion`?

Reply:
Im little affraid that its standardized naming, I saw it used in every example of webhooks that mutates to different version.